packages/tasks-gen/scripts/inference-codegen.ts

```diff
@@ -172,8 +170,10 @@ async function postProcessOutput(path2generated: string, outputSpec: Record<stri
 	return;
 }
 
-const rootDir = rootDirFinder();
+const rootDir = path.join(rootDirFinder(), "..", "tasks");
```

Comment:
Honestly we could just do `const rootDir = "../tasks"` since the script is always executed in the `/packages/tasks-gen` context